homeassistant/components/homekit_controller/connection.py

```diff
@@ -438,31 +429,18 @@ async def async_unload(self) -> None:
             self.config_entry, self.platforms
         )
 
-    def async_notify_config_changed(self, config_num: int) -> None:
-        """Notify the pairing of a config change."""
-        self.pairing.notify_config_changed(config_num)
-
     def process_config_changed(self, config_num: int) -> None:
         """Handle a config change notification from the pairing."""
         self.hass.async_create_task(self.async_update_new_accessories_state())
 
     async def async_update_new_accessories_state(self) -> None:
         """Process a change in the pairings accessories state."""
-        self.async_save_entity_map()
         await self.async_process_entity_map()
         if self.watchable_characteristics:
             await self.pairing.subscribe(self.watchable_characteristics)
         await self.async_update()
         await self.async_add_new_entities()
 
-    @callback
-    def async_save_entity_map(self) -> None:
```

Comment:
If aiohomekit is properly calling _update_accessories_state_cache everywhere we don't need this.